tests/test_pipelines.py

```diff
@@ -336,17 +336,30 @@ def test_Pipeline__get_peak_detection_info__passes_twitches_point_up_parameter_w
         pipelines, "peak_detector", autospec=True
     )
     pipeline.get_peak_detection_results()
-    mocked_peak_detection.assert_called_once_with(ANY, twitches_point_up=False)
+    mocked_peak_detection.assert_called_once_with(ANY, twitches_point_up=True)
 
 
-def test_Pipeline__get_peak_detection_info__passes_twitches_point_up_parameter_when_true(
+def test_Pipeline__get_peak_detection_info__passes_force_data_parameter_when_true(
     mocker, raw_generic_well_a1
 ):
-    pt = PipelineTemplate(100, magnetic_twitches_point_up=True)
+    pt = PipelineTemplate(100, is_force_data=True)
     pipeline = pt.create_pipeline()
     pipeline.load_raw_magnetic_data(raw_generic_well_a1, raw_generic_well_a1)
     mocked_peak_detection = mocker.patch.object(
         pipelines, "peak_detector", autospec=True
     )
     pipeline.get_peak_detection_results()
     mocked_peak_detection.assert_called_once_with(ANY, twitches_point_up=True)
+
+
+def test_Pipeline__get_peak_detection_info__passes_force_data_parameter_when_false(
+    mocker, raw_generic_well_a1
+):
+    pt = PipelineTemplate(100, is_force_data=False)
+    pipeline = pt.create_pipeline()
+    pipeline.load_raw_magnetic_data(raw_generic_well_a1, raw_generic_well_a1)
+    mocked_peak_detection = mocker.patch.object(
+        pipelines, "peak_detector", autospec=True
+    )
+    pipeline.get_peak_detection_results()
+    mocked_peak_detection.assert_called_once_with(ANY, twitches_point_up=False)
```

Comment:
Just a question because I've never seen this before, does this cause `assert_called_once_with` to ignore all the other args (both positional and kwargs) and only look for `twitches_point_up=False`?

Reply:
I'm not sure actually. I used what was done in the previous test where it was asserted that twitches_point_up was true to test the opposite case where twitches_point_up is false.

Reply:
I think `ANY` here will only ignore the first positional arg

```python
>>> mock = Mock(return_value=None)
>>> mock(1, 2, test=3)

>>> mock.assert_called_once_with(ANY, test=3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jason/.pyenv/versions/3.8.7/lib/python3.8/unittest/mock.py", line 925, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "/Users/jason/.pyenv/versions/3.8.7/lib/python3.8/unittest/mock.py", line 913, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: expected call not found.
Expected: mock(<ANY>, test=3)
Actual: mock(1, 2, test=3)

```